mobile/lib/http/test_list.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+class TestList {
+  final List testList;
+
+  const TestList({
+    required this.testList,
+  });
+
+  factory TestList.fromJson(Map<String, dynamic> json) {
+    return TestList(
+      testList: json['data'],
+    );
+  }
+}
+
+Future<TestList> fetchTestList() async {
+  final response = await http.get(Uri.parse('http://127.0.0.1:3000/api/v1/tests'));
```

Comment:
darn I think i left the render in a broken state, but here's the url:
https://recommendations-backend-h7dq.onrender.com/api/v1/tests